torch/csrc/autograd/profiler.cpp

```diff
@@ -22,19 +22,24 @@ void registerCUDAMethods(CUDAStubs* stubs) {
 
 ProfilerState state = ProfilerState::Disabled;
 uint16_t next_thread_id = 0;
-std::mutex all_event_lists_mutex;
-std::list<std::shared_ptr<RangeEventList>> all_event_lists;
+std::mutex all_event_lists_map_mutex;
```

Comment:
nit: this mutex also protects `next_thread_id` right?

Reply:
Right, let me update the name to indicate so.